arbitrator/stylus/src/test/api.rs

```diff
@@ -33,9 +34,12 @@ impl TestEvmApi {
         let mut storage = HashMap::new();
         storage.insert(program, HashMap::new());
 
+        let mut transient_storage = HashMap::new();
+
         let api = TestEvmApi {
             contracts: Arc::new(Mutex::new(HashMap::new())),
             storage: Arc::new(Mutex::new(storage)),
+            transient_storage: Arc::new(Mutex::new(transient_storage)),
```

Comment:
Let's drop this and related funcs until we have a test framework with a notion of transactions (all tests are currently just isolated calls, so there's no meaningful distinction between normal and transient storage)

Reply:
:+1: